.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+### JIRA link (if applicable)
```

Comment:
Can we remove this heading and just keep the text below?

Reply:
Agree - changes made 👍